third_party/2and3/click/decorators.pyi

```diff
@@ -10,7 +10,7 @@ _F = TypeVar('_F', bound=Callable[..., Any])
 # _Decorator = Callable[[_F], _F]
 
 _Callback = Callable[
-    [Context, Union[Option, Parameter], Union[bool, int, str]],
+    [Context, Union[Option, Parameter], value: Union[Any]],
```

Comment:
The idea is correct, but the syntax is not. The `value:` part does not work in Python and the `Union` with only one element is redundant. This should work:

```suggestion
    [Context, Union[Option, Parameter], Any],
```

Reply:
Thanks, on it.